helm/examples/azure/hpcc-azurefile/values.yaml

```diff
@@ -3,6 +3,7 @@ common:
   mountPrefix: "/var/lib/HPCCSystems"
   #secretName: "azure-secret"
   #secretNamespace: "default"
+  provisioner: "file.csi.azure.com" # Set to "kubernetes.io/azure-file" if not use the csi driver but it is not recommended.
```

Comment:
see last review comment (https://github.com/hpcc-systems/HPCC-Platform/pull/16087#discussion_r872353911)
should say "if not using"

Reply:
Fixed